ais/models.py

```diff
@@ -852,6 +863,10 @@ class AddressSummary(db.Model):
     geocode_type = db.Column(db.Text)
     geocode_x = db.Column(db.Float)
     geocode_y = db.Column(db.Float)
+    # geocode_curb_x = db.Column(db.Float)
+    # geocode_curb_y = db.Column(db.Float)
+    geocode_street_x = db.Column(db.Float)
```

Comment:
i see where you're coming from, and yes we don't want to take a performance hit over this. i think part of this goes back to what our ideal API is. `in_street` as a flag seems random to me -- it's really specifying one specific type of parcel geocode. i think it would be more useful long-term to have a parameter like `/addresses/1234 market st?parcel_geocode_location=[street|curb|centroid]`, in addition to a param for which geocode types are allowable (e.g. `/addresses/1234 market st?geocode_types=pwd_parcel;true_range`). from that standpoint, adding the `geocode_street_x/y` to address summary isn't an general solution. based on that strategy we would need to add _all_ geocode types to address summary, which seems messy for a table that's meant as a concise reference for people to browse/bring into arcmap/interact with. that's why i would prefer to use the geocode table until we know that there really is a performance issue that can't be solved with better indexing, etc. if you want to leave a `TODO` for that and move ahead with this that's totally fine, just didn't want to lose sight of the farther-out vision for this.